omod/src/main/java/org/openmrs/module/pihapps/rest/converter/PihAppsBeanConverter.java

```diff
@@ -0,0 +1,89 @@
+package org.openmrs.module.pihapps.rest.converter;
+
+import org.apache.commons.beanutils.PropertyUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.annotation.Handler;
+import org.openmrs.module.pihapps.PihAppsConfig;
+import org.openmrs.module.pihapps.orders.LabOrderConfig;
+import org.openmrs.module.pihapps.orders.LabTestCategory;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.web.ConversionUtil;
+import org.openmrs.module.webservices.rest.web.representation.CustomRepresentation;
+import org.openmrs.module.webservices.rest.web.representation.Representation;
+import org.openmrs.module.webservices.rest.web.resource.api.Converter;
+import org.openmrs.module.webservices.rest.web.resource.impl.DelegatingResourceDescription;
+import org.openmrs.module.webservices.rest.web.response.ConversionException;
+import org.openmrs.module.webservices.rest.web.response.ResourceDoesNotSupportOperationException;
+
+import java.beans.PropertyDescriptor;
+import java.util.Map;
+
+@Handler(supports = {
+        PihAppsConfig.class,
+        LabOrderConfig.class,
+        LabTestCategory.class
+}, order = 0)
+public class PihAppsBeanConverter<T> implements Converter<T> {
```

Comment:
This follows the pattern of something I added to emrapi to enable retrieving the emrapiProperties class RESTfully and supporting either all properties or a custom representation that just evaluates the properties being requested.  This allows us to expose our configuration classes restfully and get the properties we need for a given page deliberately.